Brew update v1.3.3 ChecksumMismatchError

## What did you do?
```brew upgrade carting```

## What did you expect to happen?
- Successfully upgrade to v1.3.3

## What happened instead?

```
Error: An exception occurred within a child process:
  ChecksumMismatchError: SHA256 mismatch
```

@artemnovichkov Thanks for getting back to me.  Unfortunately, I'm still getting an error (see below).      Weird thing is "brew cleanup has not been run in 30 days".  I ran it just before attempting to upgrade your package.  NOTE: I cleared the cache and double checked the cache folder, which was empty.

```bash

Updating Homebrew...
==> Upgrading 1 outdated package:
artemnovichkov/projects/carting 1.3.2 -> 1.3.3
==> Upgrading artemnovichkov/projects/carting 
==> Downloading https://github.com/artemnovichkov/Carting/archive/1.3.3.tar.gz
==> Downloading from https://codeload.github.com/artemnovichkov/Carting/tar.gz/1.3.3
######################################################################## 100.0%
==> make install PREFIX=/usr/local/Cellar/carting/1.3.3
Last 15 lines from /Users/.../Library/Logs/Homebrew/carting/01.make:
      _$ss12_ArrayBufferV19_getElementSlowPathyyXlSiF5Files18FileSystemIteratorCyAD6FolderCG_Tg5Tm in Files.swift.o
  "_swift_updateClassMetadata2", referenced from:
      _$s5Files10FileSystemC4ItemCMr in Files.swift.o
      _$s5Files4FileCMr in Files.swift.o
  "_swift_willThrow", referenced from:
      _$s11CartingCore27FrameworkInformationServiceC010frameworksD0SayAA0cD0VGyKF in FrameworkInformationService.swift.o
      _$s11CartingCore27FrameworkInformationServiceC12updateScript8withName4path6formatySS_SSSgAA9ArgumentsV6FormatOtKFyAA6TargetCKXEfU0_Tf4nnnxnn_n in FrameworkInformationService.swift.o
      _$s11CartingCore27FrameworkInformationServiceC12updateScript8withName4path6formatySS_SSSgAA9ArgumentsV6FormatOtKFTf4xnnn_n in FrameworkInformationService.swift.o
      _$s11CartingCore17FrameworksServiceC13scriptsString33_2611F9B984D3534A5E872CDF82536EAELL011fromProjectF0SnySS5IndexVG_SStSS_tKFTf4xd_n in FrameworksService.swift.o
      _$s11CartingCore14ProjectServiceC7projectyAA0C0CSSSgKF in ProjectService.swift.o
      _$s11CartingCore14ProjectServiceC6updateyyAA0C0CKFTf4nd_n in ProjectService.swift.o
      _$s11CartingCore14TargetsServiceC13targetsString33_0DEEBEB5AD2A02368FD70DDDEA86198FLL011fromProjectF0SnySS5IndexVG_SStSS_tKFTf4xd_n in TargetsService.swift.o
      ...
ld: symbol(s) not found for architecture x86_64
make: *** [build] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/artemnovichkov/homebrew-projects/issues


==> `brew cleanup` has not been run in 30 days, running now...

```

I'm getting the following error:
```bash
==> make install PREFIX=/usr/local/Cellar/carting/1.3.4
Last 15 lines from .../Library/Logs/Homebrew/carting/01.make:
make
install
PREFIX=/usr/local/Cellar/carting/1.3.4

swift build --disable-sandbox -c release
Fetching https://github.com/johnsundell/files.git
Fetching https://github.com/johnsundell/shellout.git
error: dependency graph is unresolvable; found these conflicting requirements:

Dependencies: 
    https://github.com/johnsundell/files.git @ 3.0.0..<4.0.0
'Carting' /private/tmp/carting-20190402-9250-72bkdj/Carting-1.3.4: warning: Ignoring duplicate product 'Carting' 
'Carting' /private/tmp/carting-20190402-9250-72bkdj/Carting-1.3.4: error: product dependency 'Files' not found
'Carting' /private/tmp/carting-20190402-9250-72bkdj/Carting-1.3.4: error: product dependency 'ShellOut' not found
make: *** [build] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/artemnovichkov/homebrew-projects/issues
```

--- a/Package.resolved
+++ b/Package.resolved
@@ -3,16 +3,16 @@
     "pins": [
       {
         "package": "Files",
-        "repositoryURL": "https://github.com/johnsundell/files.git",
+        "repositoryURL": "https://github.com/JohnSundell/Files.git",
         "state": {
           "branch": null,
-          "revision": "7de5a7b9627e8f2f88eba8a899ce8ddf2c64d39d",
-          "version": "3.0.0"
+          "revision": "a84615f4558151fab52ac38df697ce2442991f93",
+          "version": "2.3.0"
         }
       },
       {
         "package": "ShellOut",
-        "repositoryURL": "https://github.com/johnsundell/shellout.git",
+        "repositoryURL": "https://github.com/JohnSundell/ShellOut.git",
         "state": {
           "branch": null,
           "revision": "d3d54ce662dfee7fef619330b71d251b8d4869f9",

--- a/Package.swift
+++ b/Package.swift
@@ -9,8 +9,8 @@ let package = Package(
         .library(name: "Carting", targets: ["CartingCore"])
     ],
     dependencies: [
-        .package(url: "https://github.com/johnsundell/files.git", from: "3.0.0"),
-        .package(url: "https://github.com/johnsundell/shellout.git", from: "2.0.0")
+        .package(url: "https://github.com/JohnSundell/Files.git", from: "2.3.0"),
+        .package(url: "https://github.com/JohnSundell/ShellOut.git", from: "2.0.0")
     ],
     targets: [
         .target(name: "Carting", dependencies: ["CartingCore"]),

